packages/core/test/sources/metacoin/contracts/Migrations.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.25 <0.7.0;
+
+contract Migrations {
+  address public owner;
+  uint public last_completed_migration;
+
```

Comment:
What do you think of eliminating the Migration contract from all but one unit test? It's another optimization, no?

Reply:
If it exists we use it, and ignore it if they are missing

Reply:
I ended up changing this to an empty contract because there is some behavior requiring 3 contracts.

Reply:
Oh, I didn't realize contract counts were needed. cool.